apple/swift-docc-render
# Update svg asset for collection icon

Bug/issue #, if applicable: 92136880

## Summary

Updates the design of the SVG icon for "collection" topics.

Example:
<img width="1459" alt="Screen Shot 2022-04-22 at 11 23 00 AM" src="https://user-images.githubusercontent.com/212918/164772677-728b4587-06e6-4140-a19e-a75dee59f7bc.png">

## Testing

Steps:
1. Run `VUE_APP_DEV_SERVER_PROXY=https://ethan-kusters.github.io/swift-argument-parser npm run serve`
2. Open http://localhost:8080/documentation/argumentparser/argument and verify that the sidebar icon for "CustomStringConvertible Implementations" looks updated and matches the one in the "Topics" section.
3. Click the "CustomStringConvertible Implementations" link and verify that the intro element icon looks updated as well.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] ~~Added tests~~ (SVG only change)
